desktop/libresprite.desktop

```diff
@@ -4,7 +4,7 @@ Encoding=UTF-8
 Name=LibreSprite
 GenericName=Sprite Editor
 Comment=Animated sprite editor & pixel art tool
-Icon=libresprite
+Icon=ase
```

Comment:
I'm not sure this will work for system-wide install (`make install`), but as I mentioned it looks like we don't install icons right now anyway, so we can look into this later.

The icons should likely be renamed to `libresprite.png` anyway, or if they're used for something else in-editor, there could be a duplicate `libresprite.png` high-res icon (or `libresprite.svg`) for packaging purposes.

Reply:
I feel renames and correct installation packaging should be handled in a separate PR. The purpose of this is to allow Travis CI itself to create an AppImage based on the current project and deploy it to a GitHub release for download.

This same concept applies to the prior comment relating to updating g++, though you also mentioned that is for another PR.

Reply:
I'm working on a PR to fix the icons install.

Reply:
Done in #46, so you can undo this change to the desktop file, and install one of the `desktop/icons/hicolor/*/apps/libresprite.png` in the AppImage (or several of them if it supports multiple icon sizes for different uses).